### PR TITLE
media-libs/oyranos: 0.9.6 version bump, drop USE=qt4

### PR DESCRIPTION
**Pull request by a17r**
@johu 

(PR with patches is currently pending upstream)

### CODE REVIEW

File: media-libs/openicc/openicc-0.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+)
+
+S="${WORKDIR}/config-${PV}"
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DENABLE_STATIC_LIBS=$(usex static-libs)
+		-DENABLE_TESTS=$(usex test)
+		$(multilib_is_native_abi || echo -DENABLE_TOOLS=OFF )
+		-DCMAKE_INSTALL_DOCDIR=/usr/share/doc/${PF}
```
Comment (SoapGentoo) [inline media-libs/openicc/openicc-0.1.0.ebuild]: `GNUInstallDirs` should not be used with absolute paths (will kill Prefix), `-DCMAKE_INSTALL_DOCDIR=share/doc/${PF}` should suffice

File: media-libs/openicc/openicc-0.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		-DCMAKE_DISABLE_FIND_PACKAGE_Doxygen=$(usex !doc)
```
Comment (SoapGentoo) [inline media-libs/openicc/openicc-0.1.0.ebuild]: don't you think `-DCMAKE_DISABLE_FIND_PACKAGE_Doxygen=$(multilib_native_usex doc OFF ON)` is a better fit? i.e. only build docs in the native ABI

File: media-libs/oyranos/oyranos-9999.ebuild [hunk 3]
```diff
@@ -48,9 +65,9 @@ RDEPEND="${COMMON_DEPEND}
 	media-libs/icc-profiles-basiccolor-printing2009
 	media-libs/icc-profiles-openicc"
 
-REQUIRED_USE="qt4? ( X ) qt5? ( X )"
+REQUIRED_USE="qt5? ( X )"
 
-DOCS=( AUTHORS.md ChangeLog.md README.md )
+DOCS=( {AUTHORS,ChangeLog,README}.md )
```
Comment (SoapGentoo) [inline media-libs/oyranos/oyranos-9999.ebuild]: are any of these needed and not already covered by EAPI 6 `einstalldocs`?

File: media-libs/openicc/openicc-0.1.0.ebuild (hunk 1 above)
Comment (a17r) [inline media-libs/openicc/openicc-0.1.0.ebuild]: thanks for the input. Btw GNUInstallDirs patch already upstreamed ;)

**Comment by a17r** [comment]
Patches have been accepted upstream, so live ebuild will work as well once this is merged.

**Comment by johu** [comment]
Thanks

**Pull request closed without merging**